lib/evmone/instruction_traits.hpp

```diff
@@ -1,12 +1,20 @@
 // evmone: Fast Ethereum Virtual Machine implementation
 // Copyright 2020 The evmone Authors.
 // SPDX-License-Identifier: Apache-2.0
+#pragma once
 
 #include <evmc/instructions.h>
 #include <array>
 
 namespace evmone::instr
 {
+/// EIP-2929 constants (https://eips.ethereum.org/EIPS/eip-2929).
+inline constexpr auto cold_sload_cost = 2100;
```

Comment:
Not sure what we're going to do with these constants if they change in future forks... Mabye just prepend with `berlin_`. But I guess it's ok to leave like this now.

Reply:
True. But it would be easy to move the to constexpr array. E.g. `instr::constants[EVMC_BERLIN].cold_sload_cost`.